recipes/tensorflow-text/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.17.0" %}
+
+package:
+  name: tensorflow-text
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
+  sha256: 039f05f422a3ad55289a4897b5427d07cac7efe3943927c3b3c7a593f6123989
+
+build:
+  number: 0
+  skip: true  # [win]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+
+requirements:
+  build:
+  - {{ compiler('c') }}
+  - {{ stdlib('c') }}
+  - {{ compiler('cxx') }}
+  - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
+  - bazel >=5.2.0
+  host:
+  - python
+  - pip
+  - packaging
+  - setuptools >=70
+  - wheel
+  - absl-py
+  - auditwheel
+  - patchelf
+  # This ensures that a consistent version of openssl is chosen between
+  # all packages.
+  # https://github.com/conda-forge/conda-forge.github.io/issues/1528
+  - openssl
+  - {{ pin_compatible('tensorflow', max_pin='x.x') }}
```

Comment:
```suggestion
  - tensorflow
```

Reply:
It needs to be within the same minor version.

Reply:
update the suggestion, then you should do something like

```
{{ set version_major_minor = ".".join(version.split(".")[:2]) }}
```

The pin compatible syntax doesn't do what you think it does.

Reply:
Done.